contracts/rewards/src/contract/execute.rs

```diff
@@ -160,11 +160,11 @@ pub fn create_pool(
 
 pub fn update_pool_params(
     storage: &mut dyn Storage,
-    pool_id: &PoolId,
+    pool_id: PoolId,
```

Comment:
I don't understand this change. Why own the PoolId if you only need it for references?

Reply:
fixed